semantic_conventions/lib/opentelemetry/semantic_conventions/version.rb

```diff
@@ -6,6 +6,6 @@
 
 module OpenTelemetry
   module SemanticConventions
-    VERSION = '1.10.0'
+    VERSION = '2.0.0'
```

Comment:
This is problematic. Until now, we have had this version track the corresponding semantic conventions spec version. I think we should try to continue with that, if possible. TBH, part of me doesn't want to bump this at all.

Reply:
This should be sufficient to not release it I believe 47cfc77fff092488da3b11dd92c7aee518f60be9